HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -739,13 +732,15 @@
 
   <sch:pattern name='[MechanicalVentilationType=SharedWithPreHeating]'>
     <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:MechanicalVentilation/h:VentilationFans/h:VentilationFan[h:UsedForWholeBuildingVentilation="true" and h:IsSharedSystem="true"]/h:extension/h:PreHeating'>
+      <sch:assert test='count(../../h:FanType[text()="exhaust only"]) = 0'>Expected 0 element(s) for xpath: ../../FanType[text()="exhaust only"]</sch:assert>
```

Comment:
Probably not an issue in the context, but will it disable any exhaust fan (including in-unit exhaust) once there's one preconditioned fan?

Reply:
Given the context, it should only look at the fan type of a shared ventilation fan with preheating. But feel free to test it and double-check.

Reply:
Just realized the base-mechven-shared-preconditioning.xml already did that.